scenes/lessons/7th_class/a_b+c+d/a_b+c+d.py

```diff
@@ -0,0 +1,616 @@
+from manim import WHITE, YELLOW, ORANGE, BLUE, RED, MAROON, GOLD_A, TEAL, GREEN
+from manim import MathTex, VGroup
+from manim import SurroundingRectangle, Rectangle, Circle
+from manim import RIGHT, LEFT, UP, DOWN, DR, UL, DR, ORIGIN
+from manim import AnimationGroup, Write, Unwrite, FadeIn, FadeOut, Create, ReplacementTransform
+from manim import Indicate, Circumscribe, Wiggle, TransformMatchingTex, ClockwiseTransform
+
+import numpy as np
+from colour import Color
+
+from aramanim import ConectionLine
+from hanrahashiv import FormulaModificationsScene, ModifyFormula
+
+BIG_SCALE = 1.3
+SCALE = .45
+
+
+def get_tex(expression, color=WHITE):
+    """
+    Receives a math expression, and attempts to parse that into a Tex object.
+    It's pretty basic and the main advantage is the readability of the expression.
+    Avoids the dollar signs and other ugly details.
+    WARNING: Don't use this for very complex formulas.
+    """
+    tex_list = list()
+    if '$\cdot$' in expression:
+        raise Exception("Please provide the dot multiply sign as '*'")
+    skip_char = False
+    for i, char in enumerate(expression):
+        if skip_char:
+            skip_char = False
+            continue
+        if char.isalpha():
+            tex_list.append(char)
```

Comment:
Do we ever use variable names with length > 1?

Reply:
Ինձ թվում կարա լինի։ Ես լատեքսին խորը ծանոթ չեմ, բայց արմատ, լոգարիթմ և այլ կոմպլեքս ֆունկցիաները պետք ա որ էդ լոգիկայով աշխատեն։ Էս դեպքում կարդալու հեշտության համար որոշեցի գրեմ էդ ֆունկցիան, որովհետև գիտեի որ կոմպլեքս բաներ չկան էս խնդրում։